main.go

```diff
@@ -42,6 +42,7 @@ func main() {
 
 	ui := &cli.ColoredUi{
 		ErrorColor: cli.UiColorRed,
+		WarnColor:  cli.UiColor{Code: int(color.FgYellow), Bold: false},
```

Comment:
cli's yellow uses `color.FgHiYellow` which doesn't work in my terminal at least 🤔 got it around by doing this manually